src/test/net_tests.cpp

```diff
@@ -301,12 +301,18 @@ BOOST_AUTO_TEST_CASE(cnetaddr_basic)
     // IPv6, scoped/link-local. See https://tools.ietf.org/html/rfc4007
     // We support non-negative decimal integers (uint32_t) as zone id indices.
     // Test with a fairly-high value, e.g. 32, to avoid locally reserved ids.
```

Comment:
nit, can drop this line as we are no longer concerned about collisions with locally reserved zone ids

Reply:
I wasn't sure whether this was a problem for parsing or for formatting, that's why I kept it. But makes sense, will remove it.